packages/platform/atoms/index.ts

```diff
@@ -1,3 +1,5 @@
 export { Booker } from "./booker/Booker";
 export { CalProvider } from "./cal-provider/index";
 export { ConnectToCal } from "./connect-to-cal-button/index";
+export * from "./src/components/ui/button";
```

Comment:
Just brainstorming: Is there a specific reason to export components used to create cal atoms and lib/utils?  If we export more and more stuff then it might be harder to understand which of the exported things are actually "real atoms" aka file is named `atoms/index.ts` but we are pushing upwards UI components used to create atoms which is just shadcn. It could be said that maybe whoever uses platform API wants to use similar button to atoms but then we are creating UI library.

Reply:
Nit picking: Also, if we export stuff here probably cleaner would be having `index.ts` file in `src/components/ui` that exports all components in UI folder, so here we would have only:
```
export * from "./src/components/ui";
```

Reply:
> Just brainstorming: Is there a specific reason to export components used to create cal atoms and lib/utils? If we export more and more stuff then it might be harder to understand which of the exported things are actually "real atoms" aka file is named `atoms/index.ts` but we are pushing upwards UI components used to create atoms which is just shadcn. It could be said that maybe whoever uses platform API wants to use similar button to atoms but then we are creating UI library.

Agreed with this point, all the UI components can easily be found from shadcn so wouldn't it better not to export the components from shadcn?